extensions/replication/ReplicationQueuePopulator.js

```diff
@@ -23,9 +23,31 @@ class ReplicationQueuePopulator extends QueuePopulatorExtension {
         if (queueEntry.getReplicationStatus() !== 'PENDING') {
             return;
         }
+
         this.publish(this.repConfig.topic,
                      `${queueEntry.getBucket()}/${queueEntry.getObjectKey()}`,
                      JSON.stringify(entry));
+
+        if (typeof queueEntry.getReducedLocations === 'function') {
```

Comment:
minor: you could simply test with `if (queueEntry.getReducedLocations) ...` (if it's there it ought to be a function)